apps/rush-lib/src/data/RushHooks.ts

```diff
@@ -0,0 +1,20 @@
+import { IRushHooksJson } from './RushConfiguration';
+import Utilities from '../utilities/Utilities';
+
+export default class RushHooks {
+  private _postCommandHooks: string[];
+
+  public constructor(rushHooksJson: IRushHooksJson) {
+    this._postCommandHooks = rushHooksJson.postCommand;
+  }
+
+  public get postCommandHooks(): string[] {
+    return this._postCommandHooks;
+  }
+
+  public onPostCommand(): void {
```

Comment:
>onPostCommand [](start = 9, length = 13)

The "on" prefix is used for event hooks that are implemented by a separate party.

Compare with a design like this:

execute(hookName: string): void